### PR TITLE
Move dev getting started guide to model developers dir

### PR DESCRIPTION
**Pull request by Pouriya-Inmanta**
# Description

* Moved the developer getting started guide to model developers directory based on my chat with Wouter *

### CODE REVIEW

**Review by bartv — APPROVED** [review]

**Review by wouterdb — APPROVED** [review]

File: changelogs/unreleased/move-developer-started-guide.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+description: Moved developer getting started guide to model developer directory
+change-type: patch
+destination-branches: [master, iso3, iso4]
+sections:
+    feature: "{{description}}"
```
Comment (wouterdb) [inline changelogs/unreleased/move-developer-started-guide.yml:6]: I don't think this requires a changelog entry.

**Review by wouterdb — CHANGES_REQUESTED** [review]
I don't think this requires a changelog entry, as the getting-started has not been released yet.

File: changelogs/unreleased/move-dev-getting-started-guide.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+sections: {}
```
Comment (wouterdb) [inline changelogs/unreleased/move-dev-getting-started-guide.yml:1]: ....

**Review by wouterdb — CHANGES_REQUESTED** [review]
changelog is empty

**Review by wouterdb — APPROVED** [review]

**Pull request merged**